packages/integrations/tailwind/README.md

```diff
@@ -146,14 +164,18 @@ module.exports = {
 }
 ```
 
-## Examples
+### Class-based directives like `dark:text-pink` with `@apply` does not work
```

Comment:
It looks like [the Tailwind docs](https://tailwindcss.com/docs/hover-focus-and-other-states) call `dark:*`, `hover:*` etc. “modifiers”. I’ve also tweaked this heading to match the previous heading format.

```suggestion
### Class with modifiers does not work with `@apply` directives
```

Reply:
The issue is that only some modifiers do not work though. `hover:` and `focus:` etc works because they generate `.class:hover`. `dark:` however generates `.dark .class` which is causing problems in scoped styles.

Maybe we should call them class-based modifiers?

Reply:
Ahhhh, thank you! I had not understood that nuance.

Then, yes, maybe this heading would be more accurate:

```suggestion
### Class-based modifiers do not work with `@apply` directives
```